ldk/javascript/examples/self-test-loop/package.json

```diff
@@ -58,7 +58,7 @@
   "dependencies": {
     "@babel/runtime": "7.13.10",
     "@babel/runtime-corejs2": "7.13.10",
-    "@oliveai/ldk": "file:../..",
```

Comment:
if we pointing to a specific version, should we move selfTestLoop out of the examples folder? I think after this change there is no benefit for it to be there.

Reply:
Possibly. I'm not quite sure where we should move it though...maybe to a different github repo or something?

Reply:
Perhaps we could chat about it to make sure we all aligned on if this could cause issues in the future. I thought selfTestLoop ideally will become an automated testing framework which will be running in CI. In this case it will need to be testing compiled version of LDK and not a specific released version. Developing locally we could override it but it wont work for CI. Perhaps publishing could point to a specific version before building the selfTestLoop, not sure. Also, I may jumping too far ahead on this topic, but want to bring it up to make sure we are aligning with a future vision.

Reply:
I guess I thought the point of this was to specifically make the self test loop reference a compiled version of the LDK, rather than a relative path. Cause the compiled version had a bug that we didn't catch cause we were using the simlink.